daemon/openrazer_daemon/hardware/mouse_mat.py

```diff
@@ -26,6 +26,27 @@ class RazerFirefly(__RazerDeviceBrightnessSuspend):
     }
 
 
+class RazerFireflyV2(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Firefly V2 (2019)
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0C04
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 19]
+    METHODS = ['get_device_type_mousemat', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'trigger_reactive_effect']
+
+    DEVICE_IMAGE = "https://d4kkpd69xt9l7.cloudfront.net/sys-master/images/h2d/h31/9296178249758/Firefly-V2-Base.png_300Wx300H"
+
+    RAZER_URLS = {
+        "top_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/images/h2d/h31/9296178249758/Firefly-V2-Base.png_300Wx300H",
+        "side_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/images/h2d/h31/9296178249758/Firefly-V2-Base.png_300Wx300H",
+        "perspective_img": "https://d4kkpd69xt9l7.cloudfront.net/sys-master/images/h2d/h31/9296178249758/Firefly-V2-Base.png_300Wx300H"
```

Comment:
You can remove the side_img & perspective_img, they'll be removed soon anyways

Reply:
@CobyBE Does this work?https://assets.razerzone.com/eeimages/support/products/1594/1594_firefly_v2.png

Reply:
Yes that works :) How did you find this image? I was also looking for that

Reply:
I find the images on the upper right corner of Razer's product support pages are usually good.  That's where I pulled this one from :)